hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -1115,6 +1125,47 @@ public void testAsyncClusteringServiceWithCompaction() throws Exception {
     });
   }
 
+  @Test
```

Comment:
Should we make this a `@ParameterizedTest` and run for different execution modes? We can validate replace commits based on execution modes, e.g. if mode is schedule then there should be no replace commits.

Reply:
nice idea, changed